loadbalancer/predicate.go

```diff
@@ -39,7 +39,7 @@ func getGroupDecision(h http.Header, group string) (string, bool) {
 		}
 	}
 
-	return "", false
+	return "", len(h[DecisionHeader]) > 0
```

Comment:
nit: I think a doc string for this function would be helpful to understand what is really going on here.